unstract/core/pyproject.toml

```diff
@@ -25,7 +25,13 @@ classifiers = [
   "Programming Language :: Python"
 ]
 # license = {text = "MIT"}
-
+[tool.pdm.dev-dependencies]
+test = [
+    "parameterized==0.9.0",
+    "pytest==8.3.3",
+    "pytest-mock",
+    "gcsfs"
```

Comment:
NIT: Consider adding versions to these to not face issues in case of updates later